src/Thinktecture.Relay.Server.Persistence.EntityFrameworkCore/StatisticsRepository.cs

```diff
@@ -82,9 +82,9 @@ public async Task ShutdownOriginAsync(Guid originId)
 		}
 
 		/// <inheritdoc />
-		public async Task CleanUpOriginsAsync(TimeSpan oldestToKeep)
+		public async Task CleanUpOriginsAsync(TimeSpan maxAge)
 		{
-			var filterTime = DateTime.UtcNow - oldestToKeep;
+			var filterTime = DateTime.UtcNow - maxAge;
 
 			_logger.LogDebug("Cleaning up statistics storage: Deleting all origins that have not been updated since {OldestOriginToKeep}.", filterTime);
```

Comment:
```suggestion
			_logger.LogDebug("Cleaning up statistics storage by deleting all origins that have not been seen since {OriginLastSeen}.", filterTime);
```